modules/xmpp/JingleSession.js

```diff
@@ -1,148 +1,161 @@
+
+import {getLogger} from "jitsi-meet-logger";
+const logger = getLogger(__filename);
+
+import * as JingleSessionState from "./JingleSessionState";
+
 /*
  * JingleSession provides an API to manage a single Jingle session. We will
  * have different implementations depending on the underlying interface used
  * (i.e. WebRTC and ORTC) and here we hold the code common to all of them.
  */
-import {getLogger} from "jitsi-meet-logger";
-const logger = getLogger(__filename);
-
-import * as JingleSessionState from "./JingleSessionState";
+export default class JingleSession {
 
-function JingleSession(me, sid, peerjid, connection,
-                       media_constraints, ice_config) {
     /**
-     * Our JID.
+     * Creates new <tt>JingleSession</tt>.
+     * @param {string} me our JID
+     * @param {string} sid the Jingle session identifier
+     * @param {string} peerjid the JID of the remote peer
+     * @param {Strophe.Connection} connection the XMPP connection
+     * @param {Object} media_constraints the media constraints object passed to
+     * the PeerConnection onCreateAnswer/Offer as defined by the WebRTC.
+     * @param {Object} ice_config the ICE servers config object as defined by
+     * the WebRTC. Passed to the PeerConnection's constructor.
      */
-    this.me = me;
+    constructor(me, sid, peerjid, connection, media_constraints, ice_config) {
+        this.me = me;
+        this.sid = sid;
+        this.peerjid = peerjid;
+        this.connection = connection;
+        this.media_constraints = media_constraints;
+        this.ice_config = ice_config;
+
+        /**
+         * Whether to use dripping or not. Dripping is sending trickle
+         * candidates not one-by-one.
+         * Note: currently we do not support 'false'.
+         */
+        this.usedrip = true;
```

Comment:
Actually the only not supported thing is changing this var from true to false. I'll remove this comment.